lib/pfconfig/objects/Net/Netmask.pm

```diff
@@ -0,0 +1,23 @@
+package pfconfig::objects::Net::Netmask;
+
+use base qw(Net::Netmask);
+
+sub new {
+    my $package = shift;
+    my $o = Net::Netmask->new(@_);
+    return bless $o, $package;
```

Comment:
It is done inconsistently some does others do not.

It has more upsides than downsides in my opinion.
However it does not change much at the end of the day.